README.md

```diff
@@ -39,6 +39,23 @@ git clone https://github.com/hyperledger-labs/fabric-smart-client.git $FSC_PATH
 
 You are ready to run the [`samples`](./samples/README.md) in `$FSC_PATH`.
 
+### Further information
```

Comment:
Shall we add a reference to the Fabric version we test against?

Reply:
We could, the version we test against is defined in the Makefile so we could either be explicit or reference the makefile. What do you think ?

Reply:
let's reference the Makefile, so that people know how to change that if they want :)